Terrain world components documentation (#1242)
o3de/o3de.org: content/docs/user-guide/components/reference/terrain/world-debugger.md

---
linktitle: Terrain World Debugger
title: Terrain World Debugger Component

> description: The Terrain World Debugger component provides a means to display a Wireframe or Bounds representaion of the Terrain World.

Reviewer: ```suggestion
description: The Terrain World Debugger component provides a means to display a wireframe or bounds representation of the Terrain World.
```

Author: Changed.

toc: true
---

> The **Terrain World Debugger** component allows a representation of the World Bounds to be shown. These can be toggled on or off.

Reviewer: ```suggestion
The **Terrain World Debugger** component allows a representation of the World Bounds to be shown. This visualization can be toggled on or off.
```

Author: Nicer, changed.

## Provider ##

[Terrain Gem](/docs/user-guide/gems/reference/environment/terrain)

## Dependencies ##

[Terrain World](/docs/user-guide/components/reference/terrain/world)

## Properties ##

> ![\[Terrain World Debugger component interface.\]](/images/user-guide/component/terrain/terrain/ui-terrain-world-debugger-A.png)

Reviewer: ```suggestion
![Terrain World Debugger component interface.](/images/user-guide/component/terrain/terrain/ui-terrain-world-debugger-A.png)
```

Author: Changed.

> ![Terrain World Debugger component interface.](/images/user-guide/component/terrain/terrain/ui-terrain-world-debugger-A.png)

Reviewer 2: New images should really be created under the proper directory structure for components - i.e., it should match the doc structure. In this case: /images/user-guide/components/reference/terrain. (Also, note that you don't need two "terrain" subdirectories.)

Reviewer 2: Also, the "ui-" prefix in the filename is unnecessary, in my opinion.

Author: Changed all three pages to reflect this.

| Property | Description | Values | Default |
| - | - | - | - |

> | Show Wireframe | Enable to show a wireframe representation of the Terrain World. | Boolean | Enabled |

Reviewer: ```suggestion
| Show Wireframe | Displays a wireframe representation of the Terrain World. | Boolean | `Enabled` |
```

Author: Changed.

> | Show World Bounds | Enable to show the Terrain World Bounds. | Boolean | Enabled |

Reviewer: ```suggestion
| Show World Bounds | Displays the Terrain World bounds. | Boolean | `Enabled` |
```

Author: Changed.
